platform/ui/src/contextProviders/getPresentationId.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Selects a presentation ID to use for this viewport.
+ * This is done to allow the same display set to be displayed more than once
+ * on screen, with different attributes such as window level and initial position.
+ * Then, when redisplaying that, the nearest/most common attribute is re-used.
+ *
+ * The attribute chosen includes the display sets, the viewport type and orientation,
+ * and a display instance counter to prevent re-using the same presentation info
+ * for more than one viewports.
```

Comment:
Please give a more detailed explanation and maybe example as to what a presentation id is and how specifically it gets created.